src/Stratis.Bitcoin.Features.PoA/ConsensusRules/PoACoinviewRule.cs

```diff
@@ -20,7 +27,12 @@ protected override bool IsProtocolTransaction(Transaction transaction)
         /// <inheritdoc/>
         public override void CheckBlockReward(RuleContext context, Money fees, int height, Block block)
         {
-            if (block.Transactions[0].TotalOut > fees)
+            Money reward = Money.Zero;
+
+            if (height == this.network.Consensus.PremineHeight)
```

Comment:
I think GetProofOfWorkReward should throw an exception saying there is no pow reward.